src/datetimepicker.android.js

```diff
@@ -66,6 +67,12 @@ export default function RNDateTimePicker(props: AndroidNativeProps) {
       break;
   }
 
+  useEffect(() => {
+    // This effect runs on unmount, and will ensure the picker is closed.
+    // This allows for controlling the opening state of the picker through declarative logic in jsx.
+    return () => (pickers[mode] ?? pickers[MODE_DATE]).close();
```

Comment:
I think we need stubs for this on iOS as well to avoid a crash

Reply:
Ahh no, this is only in the android part of the lib, right I see that now

Reply:
src/datetimepicker.android.js 

this is only loaded for android builds, datetimepicker.ios.js gets loaded for iOs.